### PR TITLE
Update rest.rst

### PR DESCRIPTION
**Pull request by kikkauz**

### CODE REVIEW

File: docs/source/admin/fs/rest.rst [hunk 1]
```diff
@@ -251,6 +251,17 @@ The field ``external`` doesn't necessarily be a flat structure. This is a more a
 
 .. attention:: Only standard :ref:`FSCrawler fields <generated_fields>` can be set outside ``external`` field name.
 
+Specifying an elasticsearch index
+~~~~~~~~~~~~~
+By default, fscrawler creates document in the index defined in the _settings.yaml (or json) file.
+However, using the REST service, it is possible to require fscrawler to use different indexes, by adding ``index=YOUR_INDEXD`` in the form data:
```
Comment (dadoonet) [inline docs/source/admin/fs/rest.rst]: `s/YOUR_INDEXD/YOUR_INDEX_NAME` ?

File: docs/source/admin/fs/rest.rst [hunk 2]
```diff
@@ -251,6 +251,17 @@ The field ``external`` doesn't necessarily be a flat structure. This is a more a
 
 .. attention:: Only standard :ref:`FSCrawler fields <generated_fields>` can be set outside ``external`` field name.
 
+Specifying an elasticsearch index
+~~~~~~~~~~~~~
```
Comment (dadoonet) [inline docs/source/admin/fs/rest.rst]: I think we need to add a blank line after this one.

Comment (dadoonet) [reply docs/source/admin/fs/rest.rst]: Also the length to `~~~~~~~~~~~~~` must be exactly the length of the text above.

So this should be:

```
Specifying an elasticsearch index
~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
```

**Comment by kikkauz** [comment]
merged - sorry about the mess

**Comment by dadoonet** [comment]
Closing as merged in #715.

**Pull request closed without merging**